index.js

```diff
@@ -114,7 +114,13 @@ module.exports = function create (opts) {
         menubar.window.setPosition(x, y)
       }
 
-      if (!opts['always-on-top']) menubar.window.on('blur', hideWindow)
+      if (!opts['always-on-top']) {
+        menubar.window.on('blur', hideWindow)
+      }
+
+      if (opts['show-on-all-workspaces'] !== false) {
```

Comment:
Electron uses `-` for options. https://github.com/atom/electron/blob/master/docs/api/browser-window.md.